addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
```

Comment:
I guess if the token is expired and we have `refreshAccessTokens` set to true it should also attempt to refresh the token there as well, and if the server rejects the refresh token then the user is unauthorised sending them back to the login form.